gateway-protocol-impl/pom.xml

```diff
@@ -27,9 +27,11 @@
   </properties>
 
   <dependencies>
+    <!-- required mostly to guarantee the protocol is processed earlier during the build -->
```

Comment:
:thinking: mostly?

Reply:
Haha, you're right, it's just plain required :smile: